packages/cli/package.json

```diff
@@ -7,17 +7,17 @@
   },
   "scripts": {
     "build": "rimraf ./build && tsc",
-    "lint": "eslint --color --ext .ts src/",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint": "eslint --color --ext .ts src/"
   },
   "dependencies": {
-    "@graphprotocol/graph-cli": "0.18.0",
-    "@graphprotocol/graph-ts": "0.18.1",
-    "@textile/hub": "0.3.3",
-    "@textile/threads-core": "^0.3.1",
     "@web3api/client-js": "0.0.1-prealpha.1",
     "@web3api/client-test-env": "0.0.1-prealpha.1",
-    "assemblyscript": "0.14.7",
+    "@web3api/schema-compose": "0.0.1-prealpha.1",
+    "@web3api/schema-bind": "0.0.1-prealpha.1",
+    "@web3api/ens-plugin-js": "0.0.1-prealpha.1",
+    "@web3api/ipfs-plugin-js": "0.0.1-prealpha.1",
+    "@web3api/ethereum-plugin-js": "0.0.1-prealpha.1",
+    "assemblyscript": "0.17.13",
```

Comment:
Yep agreed, this is because the CLI compiles your Assemblyscript modules for you. See here: https://github.com/Web3-API/prototype/blob/c090e72b20466f3caf41a86678215a8b551bf3cd/packages/cli/src/lib/Compiler.ts#L11

In order to make the build pipeline implementation language (and compiler) agnostic, we'd need to setup a standard for projects to follow for defining their build steps. This work is starting to be tracked in these two issues:
https://github.com/Web3-API/prototype/issues/95
https://github.com/Web3-API/prototype/issues/25